README.md

```diff
@@ -1,63 +1,82 @@
 # Attention-Based Aspect Extraction
-This repository is a fork of [paper authors' repository](https://github.com/ruidan/Unsupervised-Aspect-Extraction) with
-following code improvements:
+
+This repository is a fork of [paper authors' repository](https://github.com/ruidan/Unsupervised-Aspect-Extraction) with following code improvements:
+
 * python 3 compliant
 * Keras 2 compliant
 * Keras backend independent
 
 In addition there is an additional functionality:
-* seed words 
+
+* seed words
 * no need to specify embedding dimension with external embedding usage model
 
+## Dependencies
 
-## Data
-You can find the pre-processed datasets and the pre-trained word embeddings in [[Download]](https://drive.google.com/open?id=1L4LRi3BWoCqJt5h45J2GIAW9eP_zjiNc). The zip file should be decompressed and put in the main folder.
+* keras>=2.0
+* tensorflow-gpu>=1.4
+* numpy>=1.13
 
-You can also download the original datasets of Restaurant domain and Beer domain in [[Download]](https://drive.google.com/open?id=1qzbTiJ2IL5ATZYNMp2DRkHvbFYsnOVAQ). For preprocessing, put the decompressed zip file in the main folder and run 
-```
+This code also tested to work with CNTK and MXNet. With MXNet there were some issues with Keras internals, hope they will be improved in future versions.
+
+## Data and Preprocessing
+
+You can download the original datasets (of Restaurant and Beer domains) in [[Download]](https://drive.google.com/open?id=1qzbTiJ2IL5ATZYNMp2DRkHvbFYsnOVAQ).
+
+For preprocessing, put the decompressed zip file in the main folder and run:
+```bash
 python preprocess.py
 python word2vec.py
 ```
-respectively in code/ . The preprocessed files and trained word embeddings for each domain will be saved in a folder preprocessed_data/.
+respectively in `code/`. The preprocessed files and trained word embeddings for each domain will be saved in a folder `preprocessed_data/`.
+
+You can also find the pre-processed datasets and the pre-trained word embeddings in [[Download]](https://drive.google.com/open?id=1L4LRi3BWoCqJt5h45J2GIAW9eP_zjiNc).
```

Comment:
@madrugado BTW, I don't see the "pre-trained word embeddings" there.

Reply:
there are no, they could be trained using word2vec.py

Reply:
I'm talking about the phrase: You can also find the pre-processed datasets **and the pre-trained word embeddings**.

Reply:
@madrugado, I'm talking about the phrase: You **can** also **find** the pre-processed datasets and **the pre-trained word embeddings**. But seems like we can't.